lighthouse-core/test/fraggle-rock/gather/mock-driver.js

```diff
@@ -143,12 +143,22 @@ function createMockDriver() {
   };
 }
 
-/** @param {() => jest.Mock} runProvider */
-function mockRunnerModule(runProvider) {
-  const runnerModule = {getGathererList: () => []};
-  Object.defineProperty(runnerModule, 'run', {
-    get: runProvider,
-  });
+function mockRunnerModule() {
+  const runnerModule = {
+    getGathererList: jest.fn().mockReturnValue([]),
+    gatherAndManageArtifacts: jest.fn(),
+    run: jest.fn(),
+    reset,
+  };
+
+  jest.mock('../../../runner.js', () => runnerModule);
```

Comment:
these mock commands are tricky when it comes to es modules, b/c jest hoists them to the top to make them work. https://jestjs.io/docs/manual-mocks#using-with-es-module-imports

can keep it as-is since still commonjs, or just continue to do the `jest.mock` at the toplevel but instead like

```
jest.mock('....', createRunnerModuleMock())
// not even sure if this expression with a function call can be hoisted by jest 👀 
```

no preference atm.